src/tests/simple_tests/context.rs

```diff
@@ -1,31 +1,12 @@
 use super::*;
+use crate::tests::simple_tests::asm_tests::run_asm_based_test;
 
+// TODO meta isn't checked
```

Comment:
can you explain more? 
(and if needed, create a ticket and reference it here)

Reply:
We do not check values returned by meta opcode in this test

Reply:
But I'm not sure that we want to create separate test cases for it

Reply:
EVM-659